packages/devtools_shared/lib/src/deeplink/deeplink_manager.dart

```diff
@@ -54,4 +64,122 @@ class DeeplinkManager {
       kOutputJsonField: outputJson,
     };
   }
+
+  Future<Map<String, Object?>> getAndroidAppLinkSettings({
+    required String rootPath,
+    required String buildVariant,
+  }) async {
+    final ProcessResult result = await runProcess(
+      'flutter',
```

Comment:
@DanTup I remembered you mentioned we should have a way to locate flutter in dart app. The issue https://github.com/Dart-Code/Dart-Code/issues/4722 is closed. Is there now a way to get the path flutter binary?

Reply:
Yep, see https://github.com/Dart-Code/Dart-Code/issues/4722#issuecomment-1722276722 - that issue has been implemented and is available in the current pre-release version of the extension. If you're using that. the `FLUTTER_ROOT` env variable should be available.

Let me know if this doesn't seem to be the case.